src/object/obj_internal.h

```diff
@@ -822,6 +822,37 @@ obj_dtx_need_refresh(struct dtx_handle *dth, int rc)
 	return rc == -DER_INPROGRESS && dth->dth_share_tbd_count > 0;
 }
 
+static inline void
+daos_recx_ep_list_set(struct daos_recx_ep_list *lists, unsigned int nr,
+		      daos_epoch_t eph_boundry)
+{
+	struct daos_recx_ep_list	*list;
+	struct daos_recx_ep		*recx_ep;
+	unsigned int			 i, j;
+
+	for (i = 0; i < nr; i++) {
+		list = &lists[i];
+		list->re_ep_valid = 1;
+		if (eph_boundry == 0)
+			continue;
```

Comment:
unnecessary?

Reply:
there is a call of daos_recx_ep_list_set() at cli_ec.c, that need not set eph_boundry which just pass a zero.